plugins/modules/vmware_guest.py

```diff
@@ -680,6 +680,7 @@
             type: str
             description:
             - Local administrator password.
+            - Required.
```

Comment:
```suggestion
            - If not defined, the password will be set to blank (that is, no password).
```

It's not really required, is it? But we should document that not setting this leads to a blank / empty password.

Rest LGTM.

Reply:
Forgot to fix that one. Thanks for the review.